Datadog.Serverless/Datadog.Serverless.csproj

```
@@ -0,0 +1,46 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+    <Version>0.1.0</Version>
+
+    <!-- NuGet package -->
+    <PackageId>Datadog.Serverless</PackageId>
+    <PackageReadmeFile>README.md</PackageReadmeFile>
+    <!-- <PackageIcon>packageIcon.png</PackageIcon> -->
```

You can enable this now that the icon is there.
```suggestion
    <PackageIcon>packageIcon.png</PackageIcon>
```